CMakeLists.txt

```
@@ -1073,6 +1123,7 @@ target_link_libraries(mksnapshot
     v8_initializers
     v8-bytecodes-builtin-list
     v8_torque_generated
+	-static
```

```suggestion
    -static
```
I half and half expect this to break Windows builds though.